fluent-bundle/tests/helpers/mod.rs

```diff
@@ -88,7 +88,7 @@ pub fn get_defaults(path: &str) -> Result<TestDefaults, io::Error> {
     Ok(serde_yaml::from_str(&s).expect("Parsing YAML failed."))
 }
 
-#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Clone)]
```

Comment:
Question: Which clippy rule is this? It seems fine since these are in tests.

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq

For tests yeah, I'd say this rule is not applicable, we can suppress the warning